server/cluster/cluster.go

```diff
@@ -314,11 +322,13 @@ func (c *RaftCluster) Start(s Server) error {
 	if err != nil {
 		return err
 	}
+	c.checkTSOService()
```

Comment:
Can we put this func next to `checkSchedulingService`?

Reply:
prefer to check it immediately